src/app/issue-detail/timeline/items/timeline-event-added-artifact.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {CoalescedTimelineItem} from '@app/issue-detail/timeline/timeline.component';
+import {AddedArtifactEvent} from '../../../../generated/graphql-dgql';
+
+@Component({
+  selector: 'app-timeline-event-added-artifact',
+  styleUrls: ['../timeline.component.scss'],
+  template:
+    '<app-timeline-item [timelineItem]="timelineItem" [showDeleted]="!event.artifact">' +
```

Comment:
maybe use a multiline template string instead of concatenating lines with +:

```js
template:
`<app-timeline-item ...
 ...
</app-timeline-item>`
```

Reply:
Fixed everywhere